skyportal/tests/frontend/test_followup_requests.py

```diff
@@ -0,0 +1,201 @@
+import uuid
+import pytest
+import requests
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.action_chains import ActionChains
+from baselayer.app.env import load_env
+from skyportal.tests import api
+
+env, cfg = load_env()
+endpoint = cfg['app.sedm_endpoint']
+sedm_isonline = requests.get(endpoint).status_code in [200, 400]
```

Comment:
1) Put an explicit timeout on this request. If it's not up then this could hang
2) Is the base endpoint the right URL to hit or is there a https://...edu/ping endpoint?

Reply:
1. done. 
2. this is the only endpoint.